python/hail/methods/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .family_methods import trio_matrix
+from .statgen import linreg, ld_matrix
+from .qc import sample_qc
+
+__all__ = ['trio_matrix',
+           'linreg',
+           'ld_matrix',
+           'sample_qc']
```

Comment:
So I slightly worry about polluting the toplevel namespace with a bunch of method names.  How about `hf` for functions (so `hf.call_stats`) and hm for methods (so `hm.sample_qc`).  I think the alternative is to have the hail __init__ export `functions` and `methods` (fully spelled out) and let people make shorthands if they want.

Reply:
how does it work with the genetics methods + generic methods though? `hm`, `gm`?

Reply:
I totally agree that more organization is good, but thought we'd sorta settled on exposing everything through `from hail import *` for now.

Reply:
(I'm happy to back off that), especially with hail2)

Reply:
This would also make it much easier to manage imports.

Reply:
I'm also worried about dividing the expr functions between `hail.expr.functions` and `hail.genetics.expr.functions`